apps/loan/src/entities/lending.ts

```diff
@@ -29,23 +31,34 @@ type LendingSnapshotFromApi = {
   timestamp: string
 }
 
+export type LendingSnapshot = LendingSnapshotFromApi
+
 type LendingSnapshotsFromApi = {
   chain: string
   market_id: number
-  data: LendingSnapshotFromApi[]
+  data: LendingSnapshot[]
 }
 
+const getSupportedChains = memoize(async () => {
```

Comment:
Not important now but ought to use tanstack for this some day as this fetch query is rather slow and can take a few hundred milliseconds. I know it's used in the query below but would rather have it be its own standalone query.

Reply:
I'll add a todo. I can fix it in another PR